site/src/sphinx/setup-configuration.rst

```diff
@@ -47,6 +47,10 @@ defaults:
       "numMirroringThreads": null,
       "maxNumFilesPerMirror": null,
       "maxNumBytesPerMirror": null,
+      "writeQuotaPerRepository": {
+        "requestQuota" : 10,
+        "timeWindowSeconds": 1
```

Comment:
How about using default values?
I don't want the user to use these values.

Reply:
OK. The value is set to null.

Reply:
How about defining sensible default and set it here?

Reply:
Hmm... 🤔 That is a considering point. This is an optional feature and only works when turned on only by users. If users do not configure `writeQuotaPerRepository`, it will be set to null(unlimited).
We have to suggest a sensible default here, how about 5 commits/sec?

Reply:
That sounds good. 😄